src/Playwright/Core/BrowserType.cs

```diff
@@ -117,5 +122,39 @@ public async Task<IBrowserContext> LaunchPersistentContextAsync(string userDataD
                 recordHarOmitContent: options.RecordHarOmitContent,
                 recordVideo: Browser.GetVideoArgs(options.RecordVideoDir, options.RecordVideoSize)).ConfigureAwait(false)).Object;
         }
+
+        public async Task<IBrowser> ConnectAsync(string wsEndpoint, BrowserTypeConnectOptions options = null)
+        {
+            options ??= new BrowserTypeConnectOptions();
+
+            var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.SetMinimumLevel(LogLevel.Debug);
+                builder.AddFilter((f, _) => f == "PlaywrightSharp.Playwright");
+            });
+            WebSocketTransport webSocketTransport = new WebSocketTransport(wsEndpoint, options, loggerFactory);
+            await webSocketTransport.ConnectAsync().ConfigureAwait(false);
+            _parent.Connection.Transport = webSocketTransport;
```

Comment:
Question here. Should I be able to `Launch` and `Connect` to different browsers at the same time?

Reply:
It won't work until different playwright objects are created. Since transport is acting as a shared variable the values can be conflicting. Hence, to differentiate different playwright objects need to be created then the above case will work.

Reply:
I don't see Java, assigning the connection to the parent https://github.com/microsoft/playwright-java/blob/06b88b3d4b57e25ae021e516ff2036a9306f4624/playwright/src/main/java/com/microsoft/playwright/impl/BrowserTypeImpl.java#L78

Reply:
I tried Java way transport was not getting picked as WebTransport, default transport is set by default as StdIOTransport when `BrowserType` instance is created throughout the execution StdIOTransport was been used. Hence, had to do such a change & set WebSocketTransport when connect being used.

Reply:
Yeah, this "feels" wrong. Let's investigate why this is needed and fix the underlying cause?

Reply:
@kblok I have resolved this issue. It was from the script side I wasn't closing `Browser`. Hence the connection was not getting set properly after adding it. Now we can run both connect & launch together.